src/store/authors/actionCreators.js

```diff
@@ -0,0 +1,31 @@
+import {
+	ADD_COURSE_AUTHOR,
+	CREATE_AUTHOR,
+	REMOVE_COURSE_AUTHOR,
+	RESET_COURSE_AUTHORS,
+	SAVE_AUTHORS,
+} from './actionTypes';
+
+export const saveAuthors = (authors) => ({
+	type: SAVE_AUTHORS,
+	payload: authors,
+});
+
+export const createAuthor = (author) => ({
+	type: CREATE_AUTHOR,
+	payload: author,
+});
+
+export const addAuthor = (author) => ({
```

Comment:
Please see the comments about course authors, they shouldn't be in the store, all the logic about them should be in component

Reply:
Got it